docs/_guides/codewind-crw-quick-guide.md

```diff
@@ -0,0 +1,244 @@
+
+![Subscribe to OperatorHub](./images/codereadyworkspaces-images/CRW-OperatorHub.png){:width="900"}. 
+
+![Install CodeReady Workspaces](./images/codereadyworkspaces-images/CRW-Install.png){:width="800"}. 
+
+  * If you did not set up an auth provider, set **openShiftoAuth** to **false**. 
+  * To enable HTTPS, set **tlsSupport** to **true**.
+  * If you use a self-signed certificate, set **selfSignedCert** to **true**. 
+
+After you install the operator, continue to install CodeReady Workspaces:
+
+1\. From the OpenShift dashboard, go to `Operators`>`Installed Operators`. 
+
+2\. Select **CodeReady Workspaces**. 
+
+3\. Select the **CodeReady Workspaces** cluster then **Create CheCluster**. 
+
+4\. Configure the fields, like **tlsSupport**, as needed. 
+
+5\. Copy and paste the following yaml into the provided text box: 
+
+```yaml
+apiVersion: org.eclipse.che/v1
+kind: CheCluster
+metadata:
+  name: codeready-workspaces
+spec:
+  server:
+    cheImageTag: ''
+    cheFlavor: codeready
+    devfileRegistryImage: ''
+    pluginRegistryImage: ''
+    tlsSupport: true
+    selfSignedCert: true
+    cheWorkspaceClusterRole: 'eclipse-codewind'
+    customCheProperties:
+      CHE_INFRA_KUBERNETES_WORKSPACE__START__TIMEOUT__MIN: "15"
+      CHE_LIMITS_WORKSPACE_IDLE_TIMEOUT: "0"
+      CHE_WORKSPACE_PLUGIN__BROKER_WAIT__TIMEOUT__MIN: "15"
+  database:
+    externalDb: false
+    chePostgresHostName: ''
+    chePostgresPort: ''
+    chePostgresUser: ''
+    chePostgresPassword: ''
+    chePostgresDb: ''
+  auth:
+    openShiftoAuth: false
+    identityProviderImage: ''
+    externalIdentityProvider: false
+    identityProviderURL: ''
+    identityProviderRealm: ''
+    identityProviderClientId: ''
+  storage:
+    pvcStrategy: per-workspace
+    pvcClaimSize: 1Gi
+    preCreateSubPaths: true
+```
+  * To manually install Codewind with CodeReady Workspaces, save this yaml to a disk and run `oc apply -f <yaml>`. 
+
+6\. CodeReady Workspaces now installs and you can access the CodeReady Workspaces URL and Red Hat SSO Admin Console URL:
+
+![Install CheCluster](./images/codereadyworkspaces-images/CRW-CheCluster.png){:width="900"}. 
+
+After you install CodeReady Workspaces, log in and create a Codewind workspace from the Codewind devfile: 
+
+1\. Log in to Che. Che loads. 
+
+2\. Go to **Workspaces** then click **Add Workspace**. 
+
+3\. Click **Import Devfile**. 
+
+4\. From **Source**, click **YAML**. 
+
+5\. Go to the link, [codewind-che-plugin/0.12.0/devfile.yaml](https://raw.githubusercontent.com/eclipse/codewind-che-plugin/0.12.0/devfiles/0.12.0/devfile.yaml), then copy and paste the contents into the YAML text box in your Codewind workspace. 
+
+6\. Click **Create & Open**.
+
+### Setting up Codewind
+Because of its dependency on `buildah`, Codewind needs to run as root and privileged. To enable Codewind, run the following commands: 
+  * `oc adm policy add-scc-to-user anyuid system:serviceaccounts:<namespace>:che-workspace` 
```

Comment:
We should say "where <namespace> is the namespace that you installed CodeReady Workspaces in to"